### PR TITLE
Sign up email template

### PR DESCRIPTION
**Pull request by GeekaN2**
Priority notification of the user during registration

### CODE REVIEW

**Comment by neSpecc** [comment]
1. There should be a single paragraph of text or spaces between paragraphs
2. Password should be highlighted
3. `Create your first workspace` may be irrelevant for users that signed up after invitation. Anyway, it's better to create a separate onboarding email later.

image

File: workers/email/src/templates/emails/sign-up/html.twig [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{% block content %}
+  <tr>
+    <td>
+      <img src="{{ hostOfStatic }}/email/payment-success-icon.png" width="32" height="32" hspace="3" style="display: block; vertical-align: middle; margin: 0 auto; margin-top: 23px;">
+    </td>
+  </tr>
+  <tr>
+    <td style="padding-top: 15px; padding-right: 0; padding-left: 0; padding-bottom: 15px; text-align: center;">
+      <font color="#dbe6ff" style="font-size: 15px; color: #dbe6ff; letter-spacing: 0.4px;">
+          Thank you for registering on Hawk
```
Comment (neSpecc) [inline workers/email/src/templates/emails/sign-up/html.twig:12]:
```suggestion
          Your account has been created
```

File: workers/email/src/templates/emails/sign-up/html.twig [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      </font>
+    </td>
+  </tr>
+  <tr>
+  <td style="display: block; padding-top: 20px; padding-right: 20px; padding-bottom: 20px; padding-left: 20px; margin-bottom: 30px; border-width: 1px; border-color: #494f5e; border-style: solid; border-radius: 10px; line-height: 1.47">
+    <font color="#dbe6ff" style="font-size: 15px; letter-spacing: 0.4px;">
+      Use this password to sign in to your account and create your first workspace!<br>
```
Comment (neSpecc) [inline workers/email/src/templates/emails/sign-up/html.twig:19]:
```suggestion
      Use this password to sign in. You can change it later.
```

File: workers/email/src/templates/emails/sign-up/html.twig [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      Password: {{ password }}
+      <br>
+      To change your password click to profile image at
+      the top left corner or follow this 
+      {% include '../../components/inline-link.twig' with {href: host ~ '/account/general', label: 'link'} %}
```
Comment (neSpecc) [inline workers/email/src/templates/emails/sign-up/html.twig:24]:
```suggestion
```

**Review by nikmel2803 — APPROVED** [review]

File: workers/email/src/templates/emails/sign-up/html.twig [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    <td style="padding-top: 15px; padding-right: 0; padding-left: 0; padding-bottom: 15px; text-align: center;">
+      <font color="#dbe6ff" style="font-size: 15px; color: #dbe6ff; letter-spacing: 0.4px;">
+        Your account has been created
+      </font>
+    </td>
+  </tr>
+  <tr>
+  <td style="display: block; padding-top: 20px; padding-right: 20px; padding-bottom: 20px; padding-left: 20px; margin-bottom: 30px; border-width: 1px; border-color: #494f5e; border-style: solid; border-radius: 10px; line-height: 1.47">
+    <font color="#dbe6ff" style="font-size: 15px; letter-spacing: 0.4px;">
+      Use this password to sign in. You can change it later.<br>
```
Comment (neSpecc) [inline workers/email/src/templates/emails/sign-up/html.twig:19]:
```suggestion
      Use this password to sign in. You can change it later.<br><br>
```

**Review by ilyamore88 — APPROVED** [review]

**Pull request merged**